### PR TITLE
[DOCS] Update terminal and code snippets style

### PR DESCRIPTION
**Pull request by JessSaavedra**
- [ ] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [ ] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [ ] Code is linted - run `invoke lint` (uses `black` + `ruff`)
- [ ] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: docs/docusaurus/src/css/custom.scss [hunk 1]
```diff
@@ -95,6 +97,52 @@
...
+  font-size: var(--p-md-font-size);
+  opacity: 0.75;
+  text-transform: capitalize;
+
+  & + div {
+    position: static;
+  }
+
+  & + div > div {
+    top: calc(var(--ifm-pre-padding) / 2);
```
Comment (josectobar) [inline docs/docusaurus/src/css/custom.scss]: Can we move this to a seperate file?

File: docs/docusaurus/src/css/code_block.scss [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      width: 1em;
+    }
+
+    // Code Block Content
+    & + div {
+      position: static;
+    }
+
+    // Button group
+    & + div > div {
```
Comment (josectobar) [inline docs/docusaurus/src/css/code_block.scss]: This feels a bit fragile, is there a way we can target this differently? Maybe with the classname instead?

**Review by josectobar — CHANGES_REQUESTED** [review]
This is looking great. I just have a small comment, let me know if it's possible.

File: docs/docusaurus/src/css/code_block.scss (hunk 2 above)
Comment (JessSaavedra) [reply docs/docusaurus/src/css/code_block.scss]: We don't have a classname to target and it's not possible to wrap the component to add them, the only alternatives I can think of are `& + div > div:has(button)` or `.prism-code > div` or `.prism-code > div:has(button)`. Let me know what you think or if you have any other suggestion!

Comment (josectobar) [reply docs/docusaurus/src/css/code_block.scss]: `.prism-code > div:has(button)` feels like a less brittle 👍🏽

**Review by josectobar — APPROVED** [review]
This looks great! 👍🏽

**Pull request merged**